ReactiveSwift.playground/Pages/ActionProperty.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,138 @@
+import ReactiveSwift
+import Result
+
```

Comment:
I really like having a playground page in a PR like this. That really helps demonstrate the idea. 👏 

But I think it'd be better to create something that approximates a real-life example. So create a faux reset password example or something.